src/vs/workbench/services/positronDataExplorer/common/positronDataExplorerSummaryEnhancementsFeatureFlag.ts

```diff
@@ -25,3 +29,22 @@ export function summaryPanelEnhancementsFeatureEnabled(
 		configurationService.getValue(USE_DATA_EXPLORER_SUMMARY_PANEL_ENHANCEMENTS_KEY)
 	);
 }
+
+// Register the configuration setting to expose it in the settings UI
+const configurationRegistry = Registry.as<IConfigurationRegistry>(
+	Extensions.Configuration
+);
+configurationRegistry.registerConfiguration({
+	...positronConfigurationNodeBase,
+	scope: ConfigurationScope.MACHINE_OVERRIDABLE,
+	properties: {
+		[USE_DATA_EXPLORER_SUMMARY_PANEL_ENHANCEMENTS_KEY]: {
+			type: 'boolean',
+			default: true,
+			markdownDescription: localize(
+				'positron.dataExplorer.summaryPanelEnhancements',
+				'**CAUTION**: Enable experimental Data Explorer "Summary Panel Enhancements" feature. This feature is experimental and may not work as expected. Use at your own risk.'
```

Comment:
```suggestion
				'Enable Data Explorer "Summary Panel Enhancements" feature.'
```

Given that we are opting people _in_ to this, I think we should not so strongly couch this in warning language.